pkg/motor/query_bucket.go

```diff
@@ -1,28 +1,39 @@
 package motor
 
 import (
-	"context"
 	"net/http"
 
-	mt "github.com/sonr-io/sonr/pkg/motor/types"
+	mt "github.com/sonr-io/sonr/thirdparty/types/motor"
+	bt "github.com/sonr-io/sonr/x/bucket/types"
 )
 
-func (mtr *motorNodeImpl) QueryWhereIs(ctx context.Context, did string) (mt.QueryWhereIsResponse, error) {
-	_, err := mtr.Resources.GetWhereIs(ctx, did, mtr.Address)
+// DEPRECATED - use Query instead
```

Comment:
Did you remove all dependencies on this? If so you can just get rid of it.

Reply:
Sounds good just fixed